src/actions/write/joinNetwork.test.ts

```diff
@@ -22,7 +22,7 @@ const validInput = {
   fqdn: 'test.com',
   note: 'test-note',
   properties: stubbedArweaveTxId,
-  qty: 10,
+  qty: 10000,
```

Comment:
In this file we needed to increase our stake amount to be in line with the min amount from the global gateway contract settings